wagtail/admin/templates/wagtailadmin/pages/listing/_list.html

```diff
@@ -1,7 +1,8 @@
 {% load i18n %}
 {% load l10n %}
 {% load wagtailadmin_tags %}
-<table class="listing {% if full_width %}full-width{% endif %} {% block table_classname %}{% endblock %}">
+<div class="w-sr-only">Press 'enter' to select an item, use up and down arrows to move the item, press 'enter' to deselect the item.</div>
+<table class="listing {% if full_width %}full-width{% endif %} {% block table_classname %}{% endblock %}" aria-descrption="{% trans 'Press enter to select an item, use up and down arrows to move the item, press enter to deselect the item.' %}">
```

Comment:
Let's leave this in the table description only.
However, we need to ensure it only gets added when in 'ordering' mode.

```suggestion
<table class="listing {% if full_width %}full-width{% endif %} {% block table_classname %}{% endblock %}" {% if show_ordering_column %}aria-descrption="{% trans 'Press enter to select an item, use up and down arrows to move the item, press enter to complete the move or escape to cancel the current move action.' %}"{% endif %}>
```